src/vcbuild/msvc-dmc.d

```diff
@@ -19,6 +21,14 @@ int main(string[] args)
     newArgs ~= `/Ivcbuild`;
     newArgs ~= `/Iddmd\root`;
     newArgs ~= `/FIwarnings.h`;
+
+    if (environment.get("VisualStudioVersion", "").to!double >= 14.0)

```

Comment:
I'm not sure digger really uses the vcvarsall batches, so defaulting to version 10.0 might be better for backward compatibility. AFAIK to!double will cause an exception on empty input.

Reply:
The autotester must, I was relying on that to catch any problems. I'll change as per your comment, it's much better that way anyway.

Reply:
Thanks.

The autotester doesn't build dmd with VS, but with dmc. So no errors here will be caught.